### PR TITLE
Docs/readme

### PR DESCRIPTION
**Pull request by gustavosinbandera1**

### CODE REVIEW

**Comment by gustavosinbandera1** [comment]
I have copy and paste PR steps from Kevin contributing file, I think is very easy to follow it

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,33 +17,43 @@ to the turpial-firmware.
...
+        - [Write short functions.](#write-short-functions)
+        - [Arguments by reference.](#arguments-by-reference)
+        - [Function overloading.](#function-overloading)
+    - [Casting.](#casting)
+    - [Static functions and global functions.](#static-functions-and-global-functions)
+    - [const keyword.](#const-keyword)
+    - [Indentation.](#indentation)
+  - [Documentation styleguide](#documentation-styleguide)
 
 * [Documentation styleguide.](#documentation-styleguide)
```
Comment (jeandudey) [inline CONTRIBUTING.md]: It's repeated.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -77,24 +87,33 @@ You can contribute making a Pull-Request with the code you want to fix, or with
 the features that you would like to implement. It will follow a review process
 first, and after that it can be merged.
 
-### Making a change.
+### Start a new change
+Before you start making modifications, run these commands to create a new branch that is synchronized with dev:
 
-Antes de empezar a hacer modificaciones ejecuta estos comandos para crear una nueva rama que esté sincronizada con dev:
+    git fetch --all # Download the branches in the repository.
```
Comment (jeandudey) [inline CONTRIBUTING.md]: Use ```, indented blocks are discouraged, they're confusing.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -77,24 +87,33 @@ You can contribute making a Pull-Request with the code you want to fix, or with
...
+### Upload your changes and make Pull Request
+Once you made the changes you want to propose, perform the following steps:
+    git add -A
+    git commit -m "Here a description of your changes".
+    
+Configure your user data (You only need to enter this information the first time you push):
 
-# Synchronize with the origin branch.
-git pull origin dev 
+    git config --global user.email "tu-email@gmail.com"
```
Comment (jeandudey) [inline CONTRIBUTING.md]: use ``` blocks

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -77,24 +87,33 @@ You can contribute making a Pull-Request with the code you want to fix, or with
...
+    git config --global user.name "<tu-usuario>"
 
-# Create a branch for your feature.
-git checkout -b featurebranch 
+Push your commit to your repository fork: 
+    
+    git push featureNameYouWantToDo # upload your change to github
```
Comment (jeandudey) [inline CONTRIBUTING.md]: use ``` blocks

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -77,24 +87,33 @@ You can contribute making a Pull-Request with the code you want to fix, or with
...
+    git push -u pr
```
Comment (jeandudey) [inline CONTRIBUTING.md]: use ``` blocks

**Comment by gustavosinbandera1** [comment]
the suggested changes were done

**Comment by jeandudey** [comment]
@gustavosinbandera1 solve conflicts please.

**Comment by gustavosinbandera1** [comment]
is not needed

**Pull request closed without merging**